### DISCUSSION TITLE
Dev resources and tutorials ✋

### DISCUSSION THREAD

**Discussion by santaclauze** [ordinals/ord#2434] [General]
Hey!

After listening to the space on Twitter yesterday and the topic of dev resources, I'd like to put my hand up if possible.

I am very new to the ordinal and BTC technical sphere. I have a node running on my computer as well as the most basic aspects. However, I would like to set up a dev environment to start testing ords + recursion experiments. 

Also, I have seen that js libs have been uploaded to sats, while I understand the logic behind it, I have struggled to find any code examples or docs that explain how to call libs from other stats.

In other words, I'm a noob, I want to learn and want to create content for other noobs to increase the reach of the ordinal space.

Is there a place, discord server, IRC that combines devs that would help me achieve this? or can I get in contact with someone who has time to guide me through this so that we can quickly produce a certain amount of "getting started" tutorials.

Excited about the space, and excuse me if this is not the best place to post this!

**Comment by cryptoni9n** [comment]
Hi - here's a link to the Discord (Ordicord): https://discord.com/invite/ordinals.  I too am a noob and we really need more in the way of instruction and documentation, so I applaud your efforts of wanting to create more onboarding content.  Thanks!
